docker-compose.yml

```
@@ -0,0 +1,76 @@
+version: "3.7"
+
+networks:
+    front:
+        external:
+            name: front
+    back:
+        external:
+            name: back
+services:
+    web:
+        container_name: appbuild_nginx
+        labels:
+            traefik.frontend.rule: "Host: ${URL}"
+            traefik.port: 80
+            traefik.docker.network: "front"
+            traefik.frontend.entryPoints: http
+        image: nginx:latest
+        volumes:
+            - ./:/app:cached
+            - ./docker/nginx/api.conf:/etc/nginx/conf.d/default.conf
+        links:
+            - php
+        networks:
+            front:
+            back:
+                aliases:
+                    - "Host: back.${URL}"
+    php:
+        build: ./docker/php
+        container_name: appbuild_php_fpm
+        volumes:
+            - .:/app
+        networks:
+            back:
+            front:
+    db:
+        image: mysql:5.7
+        container_name: appbuild_mysql
+        volumes:
+            - "./.data/db:/var/lib/mysql"
+        environment:
+            MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+            MYSQL_DATABASE: ${MYSQL_DATABASE}
+            MYSQL_USER: ${MYSQL_USER}
+            MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+        networks:
+            back:
+                aliases:
+                    - ${DB_URL}
+    node:
+        build:
+            context: .
+            dockerfile: ./docker/node/Dockerfile
+        container_name: appbuild_node
+        ports:
+            - 8080:8080
+        volumes:
+            - '.:/app'
+            - '/app/node_modules'
+        networks:
+            front:
+            back:
+    reverse-proxy:
+        image: traefik:v1.6.6-alpine
+        restart: always
```

```suggestion
        restart: unless-stopped
```

I prefer to prevent unwanted restarts